Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -428,13 +413,13 @@ import BraintreeCore
 
     // MARK: - App Switch Methods
 
-    func verifyAppSwitch(with configuration: BTConfiguration, fallbackToWeb: Bool) throws -> Bool {
+        func verifyAppSwitch(with configuration: BTConfiguration) throws -> Bool {
         if !configuration.isVenmoEnabled {
             throw BTVenmoError.disabled
         }
 
 
-        if !fallbackToWeb && !isVenmoAppInstalled() {
+        if !isVenmoAppInstalled() {
```

Comment:
We can remove this if block entirely, as we have it now we would throw an error if the app is not installed, which we don't want to do since we should always fallback to web now. We can also remove the entire `isVenmoAppInstalled` method in this file.